sites/public/src/layouts/application.tsx

```diff
@@ -70,13 +73,33 @@ const Layout = (props) => {
       href: "/sign-in",
     })
   }
+  const getInMaintenance = () => {
+    let inMaintenance = false
+    const maintenanceWindow = process.env.maintenanceWindow?.split(",")
+    if (maintenanceWindow?.length === 2) {
+      const convertWindowToDate = (windowString: string) =>
+        dayjs(windowString, "YYYY-MM-DD HH:mm Z")
+      const startWindow = convertWindowToDate(maintenanceWindow[0])
+      const endWindow = convertWindowToDate(maintenanceWindow[1])
+      const now = dayjs()
+      inMaintenance = now > startWindow && now < endWindow
+    }
+    return inMaintenance
+  }
 
   return (
     <div className="site-wrapper">
       <div className="site-content">
         <Head>
           <title>{t("nav.siteTitle")}</title>
         </Head>
+        {getInMaintenance() && (
+          <div className={styles["site-alert-banner-container"]}>
+            <Alert className={styles["site-alert-banner-content"]} variant={"alert"}>
```

Comment:
I'm assuming this was intended, but when you dismiss this alert it will still reappear when you switch to a different page. I wonder if we should even allow dismissing.

Reply:
great q! i'll ask design